### DISCUSSION TITLE
About the RandCropByPosNegLabeld function and Spacingd  function

### DISCUSSION THREAD

**Discussion by quxianjiuguo** [Project-MONAI/MONAI#7281] [General]
Your MONAI framework is gorgeous. I have a question.
The slices of one medical image file is approximately 512x512x80.
I used RandCropByPosNegLabeld function to preprocess medical image file and roi size is 96x96x96.
I had to use Spacingd  function to overcome it which resulted in my results being very poor.
Is the function which called Spacingd the only way?

**Comment by KumoLiu** [comment]
Hi @quxianjiuguo, thanks for your interest here.

Do you mean that you have to Spacing to get enough slices in the third channel? You can also specify a smaller ROI, 'Resize' your data, or simply pad it.
Thanks.

(moving to discussion, feel free to create another ticket if meets any issues)
